cartographer_ros/cartographer_ros/node_options.cc

```diff
@@ -34,6 +37,14 @@ NodeOptions CreateNodeOptions(
       lua_parameter_dictionary->GetDouble("submap_publish_period_sec");
   options.pose_publish_period_sec =
       lua_parameter_dictionary->GetDouble("pose_publish_period_sec");
+  options.trajectory_publish_period_sec =
+      lua_parameter_dictionary->HasKey("trajectory_publish_period_sec")
+          ? lua_parameter_dictionary->GetDouble("trajectory_publish_period_sec")
+          : kDefaultTrajectoryPublishPeriodSec;
+  options.trajectory_hue_first =
```

Comment:
I was thinking more about the single trajectory use case, but where you have multiple configuration files for multiple robots. They are all going to be trajectory 0 when you launch Cartographer and rviz. This way, you can customize the first color (for the trajectory 0). I've removed the color stuff from this PR so we can continue that in another PR :)